CODE_OF_CONDUCT.md

```diff
@@ -75,10 +68,8 @@ This code of conduct and its related procedures also applies to unacceptable beh
 
 ## 10. Contact info
```

Comment:
I would remove `## 10 Contact info` paragraph altogether because it's contained within `##7`, it's important we don't have two places to update contact information.